modules/project_cleanup/function_source/main.go

```diff
@@ -224,6 +250,39 @@ func getCleanUpTagKeysOrTerminateExecution() bool {
 	return result
 }
 
+///r
+func getFeedsListFromEnv(envVariableName string) []string {
+	TargetIncludedFeeds := os.Getenv(envVariableName)
+	logger.Println("Try to get Cai Feeds list")
+	if TargetIncludedFeeds == "" {
+		logger.Printf("No Cai Feeds provided.")
+		return nil
+	}
+
+	var caiFeeds []string
+	err := json.Unmarshal([]byte(TargetIncludedFeeds), &caiFeeds)
+	if err != nil {
+		logger.Printf("Failed to get Cai Feeds list from [%s] env variable, error [%s]", envVariableName, err.Error())
+	} else {
+		logger.Printf("Got Cai Feeds list [%s] from [%s] env variable", caiFeeds, envVariableName)
+	}
+	return caiFeeds
+}
+
+func getCleanUpFeedsOrTerminateExecution() bool {
+	cleanUpCaiFeeds, exists := os.LookupEnv(CleanUpCaiFeeds)
+	if !exists {
+		logger.Fatalf("Clean up CaiFeeds environment variable [%s] not set, set the environment variable and try again.", CleanUpCaiFeeds)
+	}
+	result, err := strconv.ParseBool(cleanUpCaiFeeds)
+	if err != nil {
+		logger.Fatalf("Invalid Clean up CaiFeeds value [%s], specify correct value for environment variable [%s] and try again.", cleanUpCaiFeeds, CleanUpCaiFeeds)
+	}
+	return result
+}
+
+///r
```

Comment:
what is this? there are a lot o this in the code

Reply:
I added the `///r` for all block of code that I added. Since this file is quite big, it helped me to navigate by my own code. I will remove it later.